src/attributes.jl

```diff
@@ -328,7 +376,26 @@ function throw_set_error_fallback(model::ModelLike,
 end
 
 """
-    ListOfOptimizerAttributesSet()
+    submit(optimizer::AbstractOptimizer, sub::AbstractSubmittable,
+           value)::Union{Nothing, SubmittedIndex{typeof(sub)}}
+
+Submit `value` to the submittable `sub` of the optimizer `optimizer`.
+
+An [`UnsupportedSubmittable`](@ref) error is thrown if `model` does not support
+the attribute `attr` (see [`supports`](@ref)) and a [`SubmitNotAllowed`](@ref)
+error is thrown if it supports the submittable `sub` but it cannot be submitted.
```

Comment:
When should it return nothing and when should it return an index? Only if supported by the solver? Or always?